src/test/scala/nl.knaw.dans.easy.multideposit/BlackBoxSpec.scala

```diff
@@ -66,80 +92,159 @@ class BlackBoxSpec extends UnitSpec with BeforeAndAfter with MockFactory with Cu
-      val expBagInfo = expBag.resolve("bag-info.txt")
-      bagInfo.read().lines.toSeq should contain allElementsOf expBagInfo.read().lines.filterNot(_ contains "Bagging-Date").toSeq
-
-      val bagit = bag.resolve("bagit.txt")
-      val expBagit = expBag.resolve("bagit.txt")
-      bagit.read().lines.toSeq should contain allElementsOf expBagit.read().lines.toSeq
-
-      val manifest = bag.resolve("manifest-sha1.txt")
-      val expManifest = expBag.resolve("manifest-sha1.txt")
-      manifest.read().lines.toSeq should contain allElementsOf expManifest.read().lines.toSeq
-
-      val tagManifest = bag.resolve("tagmanifest-sha1.txt")
-      val expTagManifest = expBag.resolve("tagmanifest-sha1.txt")
-      tagManifest.read().lines.toSeq should contain allElementsOf expTagManifest.read().lines.filterNot(_ contains "bag-info.txt").filterNot(_ contains "manifest-sha1.txt").toSeq
-
-      val dataDir = bag.resolve("data/")
-      dataDir.toFile should exist
-      dataDir.listRecursively().map {
-        case file if Files.isDirectory(file) => file.getFileName.toString + "/"
-        case file => file.getFileName.toString
-      } should contain theSameElementsAs expectedDataContent(bagName)
-
-      managed(Files.list(bag.resolve("metadata")))
-        .acquireAndGet(_.iterator().asScala.toList)
-        .map(_.getFileName.toString) should contain only("dataset.xml", "files.xml")
-
-      val datasetXml = bag.resolve("metadata/dataset.xml")
-      val expDatasetXml = expBag.resolve("metadata/dataset.xml")
-      val datasetTransformer = removeElemByName("available")
-      datasetTransformer.transform(XML.loadFile(datasetXml.toFile)) should equalTrimmed(datasetTransformer.transform(XML.loadFile(expDatasetXml.toFile)))
-
-      val filesXml = bag.resolve("metadata/files.xml")
-      val expFilesXml = expBag.resolve("metadata/files.xml")
-      (XML.loadFile(filesXml.toFile) \ "files").toSet should equalTrimmed((XML.loadFile(expFilesXml.toFile) \ "files").toSet)
-
-      val props = settings.outputDepositDir.resolve(s"allfields-$bagName/deposit.properties")
-      val expProps = expectedOutputDir.resolve(s"input-$bagName/deposit.properties")
-      props.read().lines.toSeq should contain allElementsOf expProps.read().lines.filterNot(_ startsWith "#").filterNot(_ contains "bag-store.bag-id").toSeq
+      it should "check the files present in the bag" in {
+        doNotRunOnTravis()
+
+        managed(Files.list(bag))
+          .acquireAndGet(_.iterator().asScala.toList)
+          .map(_.getFileName.toString) should contain only(
+          "bag-info.txt",
+          "bagit.txt",
+          "manifest-sha1.txt",
+          "tagmanifest-sha1.txt",
+          "data",
+          "metadata")
+      }
+
+      it should "check bag-info.txt" in {
+        doNotRunOnTravis()
+
+        val bagInfo = bag.resolve("bag-info.txt")
+        val expBagInfo = expBag.resolve("bag-info.txt")
+
+        // skipping the Bagging-Date which is different every time
```

Comment:
I understand this as saying that you reset the system time for a little while, during which this test is ran. Correct? That assumes that time is paused while running this tests, right? (wauw, so many philosophical questions could come from that...) But does that work on every machine, even if it is very slow?
Would be interested to see some references to the parts of EASY you're referring to...